lib/vdsm/storage/lvm.py

```diff
@@ -736,10 +736,8 @@ def _updatelvs_locked(self, lvs_output, vg_name, lv_names):
         - Add new LVs to the cache.
         - Replace LVs in the cache with LVs reported by the 'lvs' command,
           updating the LV attributes.
-        - If called without lv names, remove all LVs in the VG from the cache
-          not reported by LVM.
-        - If called with lv names, remove specifed LVs in the VG from the cache
-          if they were not reported by LVM.
+        - Remove specifed LVs in the VG from the cache if they were not
+          reported by LVM.
```

Comment:
This change is complicated and not really needed. We have 3 calls to _reloadlvs():

lib/vdsm/storage/lvm.py:993:                lvs = self._reloadlvs(vgName)
lib/vdsm/storage/lvm.py:1002:            lvs = self._reloadlvs(vgName)
lib/vdsm/storage/lvm.py:1782:        _lvminfo._reloadlvs(vg, newlv)

The only call with a lv name is in renameLV:

```
def renameLV(vg, oldlv, newlv):
    log.info("Renaming LV (vg=%s, oldlv=%s, newlv=%s)", vg, oldlv, newlv)
    cmd = ("lvrename",) + LVM_NOBACKUP + (vg, oldlv, newlv)
    try: 
        _lvminfo.run_command(cmd, devices=_lvminfo._getVGDevs((vg, )))
    except se.LVMCommandError as e:
        raise se.LogicalVolumeRenameError.from_lvmerror(e)
    else:
        _lvminfo._removelvs(vg, oldlv)
        _lvminfo._reloadlvs(vg, newlv)
```

Since you remove renameLV in this PR, we have only one case, lvNames=None.

Lets first remove renameLV - before any other change. Then remove the lvNames parameter and the code handling it.

When we finish to remove the dead code we can go back to unifying the behavior of _reloadlvs/pvs.